ArduCopter/Parameters.cpp

```diff
@@ -702,7 +702,7 @@ const AP_Param::GroupInfo ParametersG2::var_info[] = {
 
     // @Param: WP_NAVALT_MIN
     // @DisplayName: Minimum navigation altitude
-    // @Description: This is the altitude in meters above which for navigation can begin. This applies in auto takeoff and auto landing.
+    // @Description: This is the altitude in meters above which for navigation can begin in auto takeoff.
```

Comment:
Scope creep: If we're going to update this parameter description how about we change to this:

DisplayName: Waypoint navigation altitude minimum
Description: Altitude in meters above which navigation will begin during auto takeoff